server/websocket.js

```
@@ -0,0 +1,229 @@
+import { createServer } from "net";
+import { v4 as uuidv4 } from "uuid";
+import { WebSocket, WebSocketServer } from "ws";
+
+const DEFAULT_PORT = 3001;
+const MAX_PORT_ATTEMPTS = 10;
+const NOTIFICATION_INTERVAL = parseInt(
+  process.env.NOTIFICATION_INTERVAL || "30000",
```

ESLint

[error] 'process' is not defined.

(no-undef)